src/script/config/dependenciesGraph.js

```diff
@@ -61,10 +60,9 @@ dependencies.set(LinkPreviewRepository, {
 dependencies.set(MediaRepository, {dependencies: [PermissionRepository], name: 'MediaRepository'});
 dependencies.set(MessageSender, {dependencies: [], name: 'MessageSender'});
 dependencies.set(PermissionRepository, {dependencies: [], name: 'PermissionRepository'});
-dependencies.set(PropertiesRepository, {dependencies: [PropertiesService, SelfService], name: 'PropertiesRepository'});
+dependencies.set(PropertiesRepository, {dependencies: [PropertiesService], name: 'PropertiesRepository'});
```

Comment:
is that correct?

Reply:
No, you are missing `SelfService` as second argument in the array of `dependencies`.